_dev/front/js/components/List/List.vue

```diff
@@ -21,7 +21,7 @@
     <ul
       class="wishlist-list"
       v-if="items.length > 0 && items"
-      v-click-outside="event => emptyPopups()"
+      v-click-outside="(event) => emptyPopups()"
```

Comment:
```suggestion
      v-click-outside="emptyPopups"
```

You should be able to do this otherwise